smolyay/basis.py

```diff
@@ -87,24 +127,23 @@ class ChebyshevFirstKind(BasisFunction):
 
     def __init__(self, n):
```

Comment:
The reason it's called n is because that's the symbol used in the mathematical notation and what is used by scipy. `deg` is used by numpy. I would prefer to use naming conventions consistent with one of these packages.

Reply:
`degree` is more descriptive than `n` or `deg`. We will probably also enable a code linter that will reject short variable names like this, so please use this. In the class documentation, you can still write something like:
```py
the `degree` *n*
```
and then use *n* in any mathematical formulas.